packages/date-picker/theme/lumo/vaadin-month-calendar-styles.js

```diff
@@ -136,18 +136,13 @@ registerStyles(
     }
 
     @media (pointer: coarse) {
-      [part~='date']:hover:not([part~='selected'])::before,
-      [part~='focused']:not([part~='selected'])::before {
+      [part~='date']:hover:not([part~='selected'])::before {
```

Comment:
I think we could use the `focus-ring` attribute set on the `vaadin-month-calendar`.

<img width="389" alt="Screenshot 2024-08-28 at 15 49 13" src="https://github.com/user-attachments/assets/fd7cdd2b-2063-4f55-8228-70f1c8a40958">

So when there is `focus-ring` set on mobile, it means user has already navigated to `vaadin-month-calendar` with the keyboard, at which point the focus outline needs to become visible. Otherwise, it can be hidden:

```css
      :host(:not([focus-ring])) [part~='focused']:not([part~='selected'])::before,
      [part~='date']:hover:not([part~='selected'])::before {
        display: none;
      }

Reply:
There was the attribute I was looking for. Thanks! I updated the other selector as well, now the blinking outline doesn't happen either.